internal/pkg/cli/pipeline.go

```diff
@@ -20,6 +20,7 @@ func BuildPipelineCmd() *cobra.Command {
 	cmd.AddCommand(BuildPipelineInitCmd())
 	cmd.AddCommand(BuildPipelineUpdateCmd())
 	cmd.AddCommand(BuildPipelineDeleteCmd())
+	cmd.AddCommand(BuildPipelineShowCmd())
```

Comment:
We probably should comment this out for now so that we don't release an empty command on next release :P